packages/template/package.json

```
@@ -0,0 +1,37 @@
+{
+  "name": "@rnx-kit/template",
+  "version": "0.0.1",
+  "description": "Template folder to use as baseline for new packages in rnx-kit",
+  "homepage": "https://github.com/microsoft/rnx-kit/tree/main/packages/template#readme",
+  "license": "MIT",
+  "files": [
+    "src/*"
```

Point to lib instead? We typically don't ship `.ts` files.
```suggestion
    "lib/*"
```